wp-content/themes/haruoshi_origin/home.php

```diff
@@ -1,5 +1,7 @@
 <?php get_header(); ?>
 
-<h2>hoge</h2>
+<div class="container">
+    hoge
+</div>
```

Comment:
一応きれいなマークアップするとこんな感じ
```suggestion
<article class="container">
    <section>
      hoge
    </section>
</div>
```